packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -526,14 +525,15 @@ export class StatsAnalyzer extends EventsScope {
           .filter((key) => key.startsWith(keyPrefix))
           .reduce((prev, cur) => prev + (this.lastStatsResults[cur]?.recv[value] || 0), 0);
 
-      if (this.meetingMediaStatus.expected.sendAudio && this.lastStatsResults['audio-send']) {
+      if (this.lastStatsResults['audio-send']) {
         // compare audio stats sent
         // NOTE: relies on there being only one sender.
         const currentStats = this.statsResults['audio-send'].send;
         const previousStats = this.lastStatsResults['audio-send'].send;
 
         if (
-          currentStats.totalPacketsSent === previousStats.totalPacketsSent ||
+          (this.lastEmittedStartStopEvent?.audio?.local === EVENTS.LOCAL_MEDIA_STARTED &&
```

Comment:
with this implementation, will we see the "No audio RTP packets sent" log if no packets are sent at all?